src/main/java/org/folio/circulation/domain/ManualBlock.java

```diff
@@ -0,0 +1,36 @@
+package org.folio.circulation.domain;
+
+import static org.folio.circulation.support.JsonPropertyFetcher.getBooleanProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getDateTimeProperty;
+import static org.folio.circulation.support.JsonPropertyFetcher.getProperty;
+
+import io.vertx.core.json.JsonObject;
+import org.joda.time.DateTime;
+
+public class ManualBlock {
+  private final JsonObject representation;
```

Comment:
I am not sure I quite agree with that as this style is used on the whole project. It might be a good idea to do the same in this class as well.

Reply:
On a PR, @marcjohnson-kint mentioned that we're going to remove coupling between domain objects and using `JsonObject`. So, I guess it's fine to use fields here instead of representation.

Reply:
@Svitlana-Zmiivska 

@bohdan-suprun is referring to some recent comments I’ve made about moving away from the style where JSON is held in the domain. This was convenient, especially when mapping to storage representations.

However, it couples the domain to both the business logic and storage API representations and mixes responsibilities. As the system grows, this becomes more painful, as it makes it harder to follow specific logic and to test smaller parts independently. 

I don’t have the bandwidth to remove all of the existing examples in one go. I’m keen to not make the burden any greater so am encouraging folks when submitting new pull requests to try to reduce the direct usage of JSON in your domain. Moving that mapping to factory methods and a `toJson` method, which isn’t ideal, but it is a start towards pushing JSON to the edges.

Does that make sense? And provide more context for what @bohdan-suprun was asking for?

Reply:
Changed